draft-ietf-quic-transport.md

```diff
@@ -2696,11 +2741,12 @@ controller.
 
 ### Response to a RST_STREAM
 
-Since streams are bidirectional, a sender of a RST_STREAM needs to know how many
-bytes the peer has sent on the stream.  If an endpoint receives a RST_STREAM
-frame and has sent neither a FIN nor a RST_STREAM, it MUST send a RST_STREAM in
-response, bearing the offset of the last byte sent on this stream as the final
-offset.
+RST_STREAM terminates a stream abruptly.  Whether any action or response can or
```

Comment:
Yeah, I think I had done a better job of that, but then this section got bitrotten due to a rewrite on the main branch.  I went back through and tried to restore that again.